notebooks/03-query/16-export-msq-query.ipynb

```diff
@@ -0,0 +1,426 @@
+   "metadata": {},
+   "source": [
+    "The next cell executes the the export:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b36cb33-1e8a-4284-966c-a3937c8a33a4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "    INSERT INTO \n",
+    "       EXTERN ( local(exportPath => '/opt/shared/exports/example-wiki-export') )\n",
+    "       AS CSV\n",
+    "    SELECT\n",
+    "      *\n",
+    "    FROM \"example-wiki-export\" \n",
+    "'''\n",
+    "display_client.run_task(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f9571de0-eac8-40f0-8970-f137aec9e767",
+   "metadata": {},
+   "source": [
+    "Take a look at the file this command created, notice that the name contains the query id, the worker task that created the file and a partition number, each MSQ worker creates one or more partitions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a1cef199-430d-4094-b006-090d1013af7b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ls /opt/shared/exports/example-wiki-export"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1580ec6e-3172-4829-97bc-53b71c72ec0e",
+   "metadata": {},
+   "source": [
+    "You can see the content of the file with:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95265824-7962-4dd6-a93f-1a6281633c53",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!head /opt/shared/exports/example-wiki-export/*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8dfe9b15-0724-419a-ad97-1a35971f1f47",
+   "metadata": {},
+   "source": [
+    "## Exporting transformed data\n",
+    "Export works for any SQL SELECT statement.\n",
+    "\n",
+    "Run following cell to export aggregate query results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2c044cdb-4631-4e3f-9552-4cedc0b598d7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!rm -rf /opt/shared/exports/example-wiki-export-agg\n",
+    "!mkdir -p /opt/shared/exports/example-wiki-export-agg\n",
+    "\n",
+    "sql='''\n",
+    "INSERT INTO \n",
+    "       EXTERN ( local(exportPath => '/opt/shared/exports/example-wiki-export-agg') )\n",
+    "       AS CSV\n",
+    "\n",
+    "SELECT \"user\" as \"user\",\n",
+    "       \"channel\" as \"channel\",\n",
+    "       SUM(\"added\"+\"deleted\") as \"total_changes\"\n",
+    "FROM \"example-wiki-export\"\n",
+    "GROUP BY 1,2\n",
+    "'''\n",
+    "display_client.run_task(sql)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9987b88a-3d10-41ac-825a-b716fd002fe6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!head /opt/shared/exports/example-wiki-export-agg/*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bbdbc0f6-11f9-4d1f-b1df-3f1af6ae2fdc",
+   "metadata": {},
+   "source": [
+    "## Controlling the size of the output files\n",
```

Comment:
Exports with multiple partitions

Suggesting this so that it neatly wraps back up into the bit above where you talk about partitions being part of the filename... Otherwise maybe just something in the body of this section will make that mental connection for peeps.

Reply:
I think that is confusing because all exports can use multiple partitions, this section is about manually controlling the size of the partitions.